helm/README.md

```diff
@@ -0,0 +1,58 @@
+# Helm suite for complete geoserver - polygon parts 
+[home](../README.md)
+* The helm provide ready to use polygon parts serving by WFS protocol
+* The initialization use side car to prepare the geoserver to be ready to use with all relevant data
+* Based on [kartoza/helm for geoserver](https://github.com/kartoza/charts/tree/develop/charts/geoserver/v0.3.3)
+* Use kartoza based geoserver images [geoserver-os from mapcolonies github](https://github.com/MapColonies/geoserver)
+
+## Main Features
+
+### Layer Auto-Config 
+* On deploy, once geoserver api is up, side-car run and create the environment.
+* Readiness detects the featureDescribe api and then release to use.
```

Comment:
Readiness will be valid when it detects the featureDescribe of the polygon parts layer

Reply:
done